cogs/user_cmds.py

```diff
@@ -310,16 +310,18 @@ def get_how_to_play_embed() -> Embed:
 ## Normal Mode
 The game is pretty simple.
 
-- Enter a word that starts with the last letter of the previous correct word.  
-- If your word is correct, the bot will react with a tick mark — :white_check_mark: .  
+- Enter a word that starts with the last letter of the previous correct word (or with the last two letters in hard mode).  
```

Comment:
I added a separate section for Hard Mode section in this embed, which makes the part within parenthesis unnecessary in my opinion.

Reply:
You are absolutely right. I think you added that just recently with the language update, right? I did not check again after merging.

On thing left it the mention of english-only letters, which is not true any more. I will take a look at that as well.

Reply:
Yes, I added the hard mode part with the language update.